packages/fxa-customs-server/test/local/email_record_tests.js

```diff
@@ -22,7 +24,7 @@ test('shouldBlock works', function(t) {
   var er = simpleEmailRecord();
 
   t.equal(er.shouldBlock(), false, 'record has never been blocked');
-  er.rl = 499;
+  er.rl = 500;
```

Comment:
This was just to tighten up the test case a bit, rather than asserting either side of the boundary assert on the boundary itself.